src/handlers/settlement-window-close.js

```diff
@@ -7,8 +7,12 @@ const handler = (router, routesContext) => {
         const { startDate: fromDateTime, endDate: toDateTime } = ctx.request.body;
 
         try {
-            await axios.post(`${routesContext.config.settlementManagementEndpoint}/close-window`, {});
-            ctx.response.status = 200;
+            const resp = await axios.post(`${routesContext.config.externalSettlementsEndpoint}/tmf/settlement-windows/current/close`, {});
```

Comment:
No need to add the `tmf` here. It should be already included in `externalSettlementsEndpoint`